lib/view_component/base.rb

```diff
@@ -179,9 +179,10 @@ def short_identifier
       end
 
       def inherited(child)
-        # If we're in Rails, add application url_helpers to the component context
-        if defined?(Rails)
-          child.include Rails.application.routes.url_helpers unless child < Rails.application.routes.url_helpers
+        # If Rails application is loaded, add application url_helpers to the component context
+        # we need to check this to use this gem as a dependency
+        if Rails.application
+          child.include Rails.application.routes.url_helpers
```

Comment:
```suggestion
          child.include Rails.application.routes.url_helpers < Rails.application.routes.url_helpers
```

Should we still only include the `url_helpers` if they haven't already been included?